webview/src/experiments/components/App.test.tsx

```diff
@@ -2038,7 +2038,10 @@ describe('App', () => {
     it('should not allow pushing an experiment when an experiment is running in the workspace', () => {
       renderTable()
 
-      fireEvent.click(screen.getByTestId('exp-f13bca-push-experiment'))
```

Comment:
[F] These tests are changed because I replaced the component's `data-testid` with an `aria-label`.